cli/rt/run.rs

```diff
@@ -592,6 +592,11 @@ impl ModuleLoader for EmbeddedModuleLoader {
     Some(Cow::Owned(data.data.to_vec()))
   }
 
+  fn source_map_source_exists(&self, source_url: &str) -> Option<bool> {
+    let url = Url::parse(source_url).ok()?;
+    Some(self.shared.modules.read(&url).ok().flatten().is_some())
```

Comment:
This will read from the real file system and be slow in those cases (see the error case on the `self.vfs.file_entry` call). Maybe prefer adding a `has_specifier(specifier: &Url)` method then fall back to doing a file system stat?